Docs/Design/Agents.md

```
@@ -1,6 +1,22 @@
 # Agents.md
 
 
+### Skills
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)